CONTRIBUTING.md

```diff
@@ -16,11 +16,3 @@ bundle exec rake
 VCR uses [RSpec](http://github.com/rspec/rspec) for unit tests.  The specs are written in a very "focused" style, where each spec is concerned only with exercising the object under test, using mocks as necessary.  You can run the specs using `rake spec`.
 
 [Cucumber](http://cukes.info/) is used for end-to-end full stack integration tests that also function as VCR's documentation.
-
-## Problems running bundle install?
-
-If you get an error while running `bundle install`, it may be one of the "extras" gems which are not required for development. Try installing it without these gems.
-
-```console
-bundle install --without extras
```

Comment:
🗒️ There is no `extras` group in the Gemfile, so this documentation was misleading. I removed it to prevent confusion.